src/team_position.py

```diff
@@ -0,0 +1,396 @@
+import json
+from datetime import datetime
+from collections import Counter
+
+date_now = datetime.now().strftime('%d/%b/%Y')
+
+with open('../data/group_stage.json') as file:
+    groups = json.load(file)
+
+with open('../data/match_results.json') as file:
+    match_results = json.load(file)
+
+with open('../data/group.json') as file:
+    g_name = json.load(file)
+
+
+class Match:
```

Comment:
Why do we need another Match class?

Reply:
It was added to run the code in test mode